src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -181,6 +197,24 @@ function WebAuthnCard(props) {
 		}
 	}
 
+	// Edit authenticator name
+	const onSubmitKeyName = async (values) => {
```

Comment:
@Aringoaway Imho I think the handling of the try/catch method would be better this way:

```
const onSubmitKeyName = async (values) => {
	let response;
	try {
		response = await SeaCatAuthAPI.put(`/public/webauthn/${values.id}`, {"name": `${values.name}`});
		if (response.data.result != 'OK') {
			throw new Error(t("WebAuthnScreen|Something went wrong, can't update authenticator"));
		}
		props.app.addAlert("success", t("WebAuthnScreen|Authenticator successfully updated"));
		resetField("name");
		setIsSubmitting(false);
		getAuthenticators();
	} catch(e) {
		console.error(e);
		setIsSubmitting(false);
		props.app.addAlert("danger", t("WebAuthnScreen|Something went wrong, can't update authenticator"));
	}
}
```
Because if saving the authenticator name fails, we dont want to send user to initial position, since the user can thought that everything went well (they can miss the alert message) and then wonder, why the authenticator name has not been changed.

Can you record a video with it please?

Reply:
@Pe5h4 
https://user-images.githubusercontent.com/65866093/199666019-26d03f8a-5e10-464f-8182-f8f26dd45bc5.mp4